wger/core/static/css/workout-manager.css

```diff
@@ -139,6 +139,21 @@ div.muscle-background
     background-repeat: no-repeat;
 }
 
+#svg-front {
+    display: block;
+    transform: translate(-25px, -48px)scale(0.75);
+    opacity: 0;
+
```

Comment:
Kindly reduce the spacing here for better formatting

Reply:
Will check on that.